src/util/object-as-lower-case.js

```diff
@@ -0,0 +1,2 @@
+module.exports = (obj) => Object.entries(obj)
+  .reduce((p, [h, v]) => Object.assign(p, { [h.toLowerCase()]: v }), {});
```

Comment:
`object-keys-to-lowercase` ?
Only bringing up because when I first saw this I thought it lowercased the `values` as well :)

Reply:
Changing it to `object-rekey-lower-case`